x/concentrated-liquidity/swaps.go

```diff
@@ -475,8 +475,8 @@ func (k Keeper) computeInAmtGivenOut(
 
 		// Update the swapState with the new sqrtPrice from the above swap
 		swapState.sqrtPrice = computedSqrtPrice
-		swapState.amountSpecifiedRemaining = swapState.amountSpecifiedRemaining.SubMut(amountOut)
-		swapState.amountCalculated = swapState.amountCalculated.AddMut(amountIn.Add(spreadRewardChargeTotal))
```

Comment:
fwiw, I personally find it easier to see whats going on with the equal signs

Reply:
let's bring back the equal signs